Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/ListViewExtensions.SmoothScrollIntoView.cs

```diff
@@ -198,10 +185,52 @@ public static async Task SmoothScrollIntoViewWithIndexAsync(this ListViewBase li
         /// <param name="scrollIfVisibile">Set true to disable scrolling when the corresponding item is in view</param>
         /// <param name="additionalHorizontalOffset">Adds additional horizontal offset</param>
         /// <param name="additionalVerticalOffset">Adds additional vertical offset</param>
-        /// <returns>Note: Even though this return <see cref="Task"/>, it will not wait until the scrolling completes</returns>
+        /// <returns>Returns <see cref="Task"/> that completes after scrolling</returns>
         public static async Task SmoothScrollIntoViewWithItemAsync(this ListViewBase listViewBase, object item, ScrollItemPlacement itemPlacement = ScrollItemPlacement.Default, bool disableAnimation = false, bool scrollIfVisibile = true, int additionalHorizontalOffset = 0, int additionalVerticalOffset = 0)
         {
-            await SmoothScrollIntoViewWithIndexAsync(listViewBase, listViewBase.Items.IndexOf(item), itemPlacement, disableAnimation, scrollIfVisibile, additionalHorizontalOffset, additionalVerticalOffset);
+            await SmoothScrollIntoViewWithIndexAsync(listViewBase, listViewBase.Items.IndexOf(item), itemPlacement, disableAnimation, scrollIfVisibile, additionalHorizontalOffset, additionalVerticalOffset).ConfigureAwait(true);
+        }
+
+        /// <summary>
+        /// Changes the view of <see cref="ScrollViewer"/> asynchronous.
+        /// </summary>
+        /// <param name="scrollViewer">The scroll viewer.</param>
+        /// <param name="horizontalOffset">The horizontal offset.</param>
+        /// <param name="verticalOffset">The vertical offset.</param>
+        /// <param name="zoomFactor">The zoom factor.</param>
+        /// <param name="disableAnimation">if set to <c>true</c> disable animation.</param>
+        private static async Task ChangeViewAsync(this ScrollViewer scrollViewer, double? horizontalOffset, double? verticalOffset, float? zoomFactor, bool disableAnimation)
```

Comment:
Do you guys think this method should be public? Will this extension method be useful for developers as well?

Reply:
Yeah, we could expose this. I guess the idea is if you have animation the original `ChangeView` can take longer to return?

Reply:
> I guess the idea is if you have animation the original ChangeView can take longer to return?

I am not sure what you mean exactly. If we have animation in the original `ChangeView` then it will act as fire and forget not as a synchronous call. There is no built-in way to asynchronously wait until the animation completes. `ChangeViewAsync` methods does that, it can asynchronously wait until the animation completes.

Reply:
Cool, in either case, this could be helpful. @JustinXinLiu would you use something like this?

Since this PR is good to go right now, we can always open a different one later, as we'd want to improve docs with an example maybe.